netjson-monitoring.lua

```diff
@@ -320,8 +399,28 @@ for name, interface in pairs(network_status) do
   if not is_excluded(name) and not processed[name] then
     netjson_interface = {
         name = name,
-        statistics = interface.statistics
+        type = string.lower(interface.type),
+        up = interface.up,
+        mac = interface.macaddr,
+        txqueuelen = interface.txqueuelen,
+        mtu = interface.mtu,
+        speed = interface.speed,  -- XXX: untested
+        bridge_members = interface['bridge-members'],
+        stp = interface.stp,  -- XXX: untested
     }
+    if interface.type == 'Network device' then
+        if interface['link-supported'] then
```

Comment:
we should check that this is not empty (it could be an empty table) because otherwise also openvpn tap interfaces are flagged as ethernet